feature/session/src/main/res/layout/item_session.xml

```diff
@@ -125,46 +127,46 @@
                 android:clickable="false"
                 android:ellipsize="end"
                 android:enabled="false"
-                android:text="@{session.language.getByLang(lang)}"
+                android:text="@{session.lang.text.getByLang(lang)}"
                 android:textAppearance="@style/TextAppearance.App.Caption"
-                android:textColor="@color/white"
-                app:chipBackgroundColor="#7982e1"
+                android:textColor="@{session.lang.equals(Lang.EN)?@color/blue1:@color/red1}"
+                app:chipBackgroundColor="@{session.lang.equals(Lang.EN)?@color/blue2:@color/red2}"
                 app:chipMinTouchTargetSize="0dp"
                 tools:text="@sample/service_session.json/data/language"
                 />
 
             <com.google.android.material.chip.Chip
-                android:id="@+id/simultaneous_interpretation_chip"
+                android:id="@+id/for_beginners_chip"
                 style="@style/Widget.MaterialComponents.Chip.Action"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:clickable="false"
                 android:ellipsize="end"
                 android:enabled="false"
-                android:text="@string/session_simultaneous_interpretation"
+                android:text="@string/session_for_beginners"
                 android:textAppearance="@style/TextAppearance.App.Caption"
-                android:textColor="@color/white"
-                app:chipBackgroundColor="#7982e1"
+                android:textColor="#00BE80"
```

Comment:
Please create `feature/session/src/main/res/values/colors.xml` And Can you declare it in `feature/session/src/main/res/values/colors.xml` ?
like `session_english_talk_text` 🙏